rust/xain-proto/generate.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+cd "$(dirname "$0")"
+
+if ! [[ -x "$(command -v grpc_rust_plugin)" ]]; then
+    echo "Error: grpc_rust_plugin was not found"
+    echo
+    echo "To install, run: cargo install grpcio-compiler"
```

Comment:
Maybe just run `cargo install grpcio-compiler --version x.y.z` here? That would be a nice way to pin the version of `grpcio-compiler` we are using.

Reply:
Done.